### ISSUE TITLE
R language error with more than 1 line of code in a cell.

### ISSUE THREAD

**Issue by fulky** [microsoft/vscode-jupyter#6470]
Using R integration, create a new notebook,  use the R kernel, add a code block and execute:

x <- 19
print(x)

Execution results in ...
ERROR: Error in parse(text = x, srcfile = src): <text>:1:8: unexpected input
1: x <- 19

## Environment data
Version: 1.58.0-insider (user setup)
Commit: 57930758db9e2c83f59b9ee1851400bd16192d32
Date: 2021-06-30T05:14:00.370Z
Electron: 12.0.12
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.22000

-   VS Code version:  1.58.0-insider (user setup)
-   Jupyter Extension version (available under the Extensions sidebar): v2021.7.985709665
-   Python Extension version (available under the Extensions sidebar): v2021.7.981196274-dev
-   OS (Windows | Mac | Linux distro) and version: Windows
-   Python and/or Anaconda version:3.9
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): N/A
-   Jupyter server running: Local 

## Expected behaviour

It works

## Actual behaviour

Execution results in ...
ERROR: Error in parse(text = x, srcfile = src): <text>:1:8: unexpected input
1: x <- 19

## Steps to reproduce:
Using R integration, create a new notebook,  use the R kernel, add a code block and execute:

x <- 19
print(x)

## Logs

<details>

<summary>Output for <code>Jupyter</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Jupyter</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

**Comment by IanMatthewHuff** [comment]
#6348 Looks like it might be related here. This works for me, but I'm on Mac. Per the other issue it looks like this might only repro on Windows. @rchiodo @joyceerhl @DavidKutu anyone on windows have an R kernel handy to check?

![image](https://user-images.githubusercontent.com/812783/124198317-3fc03a00-da85-11eb-940e-99803f144b43.png)

**Comment by rchiodo** [comment]
Yeah I can repro using windows. I get the same problem. I assume this is a linefeed issue with the data sent to the kernel.
